rs-permission-converter/test/permconv.test.js

```diff
@@ -0,0 +1,67 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+const fsExtra = require('fs-extra');
+const exec = require('@actions/exec');
+
+// Used by @actions/tool-cache. Directory must exist before we load module.
+const outputDir = path.join(__dirname, '..', 'test_output_dir');
```

Comment:
I think we shall try to run more tests with `mock-fs` that mocks out the filesystem.